pylint/config/deprecation_actions.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         super().__init__(
             option_strings,
             dest,
-            "+",
+            1,
```

Comment:
https://docs.python.org/3/library/optparse.html#optparse.Option.nargs

Turns out `1` is the default in `optparse`. Should probably have looked that up when I made this 😅 

This should work 😄 _It did locally..._